### DISCUSSION TITLE
forum-session-SciELO-pt

### DISCUSSION THREAD

**Discussion by giscus** [arXiv/accessibility2024#133] [General]
# forum-session-SciELO-pt

Junte-se ao SciELO e aos editores de revistas de Física e Engenharia para saber mais sobre os desafios e oportunidades na publicação de pesquisas acessíveis.

https://accessibility2024.arxiv.org/forum-session-SciELO-pt

**Comment by SBBCornell** [comment]
Bem-vindos a todos! Por favor, compartilhe pensamentos, perguntas e recursos de interesse para a comunidade.

**Comment by SBBCornell** [comment]
¡Bienvenidos a todos! Comparta pensamientos, preguntas y recursos de interés para la comunidad.
